package.json

```diff
@@ -39,6 +42,7 @@
     "eslint-plugin-react-refresh": "^0.4.7",
     "prettier": "^3.3.0",
     "prettier-plugin-solidity": "^1.3.1",
-    "rimraf": "^5.0.7"
+    "rimraf": "^5.0.7",
+    "sherif": "0.9.0"
```

Comment:
```suggestion
    "sherif": "^0.9.0"
```

Reply:
They may not be following semver right?

Reply:
Don't think that matters - we have a lock file, so the version gets pinned anyway. Also I _think_ this version _should_ follow semver (not super clear, but [the docs](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#version) say that it "must be parseable by [node-semver](https://github.com/npm/node-semver)", which itself is a semantic versioning package for NodeJS).